man/ru/apol.1

```diff
@@ -0,0 +1,43 @@
+.\" Copyright (c) 2016 Tresys Technology, LLC.  All rights reserved.
+.TH apol 1 2016-02-20 "Tresys Technology, LLC" "SETools: утилиты анализа политики SELinux"
+
+.SH ИМЯ
+apol \- графическая утилита анализа политики SELinux
+
+.SH ОБЗОР
+\fBapol\fR [OPTIONS] [POLICY]
+
+.SH ОПИСАНИЕ
+.PP
+\fBapol\fR - графическая утилита, которая позволяет пользователю изучить и проанализировать аспекты политики SELinux.
+
+.SH ПОЛИТИКА
+.PP
+\fBapol\fR поддерживает загрузку политик SELinux в одном из двух форматов.
+.RS
+.IP "source:"
+Один текстовый файл, содержащий источник монолитной политики. Этот файл обычно называется policy.conf.
+.IP "binary:"
+Один файл, содержащий двоичную политику. В системах Linux название этого файла обычно соответствует версии, например, \fIpolicy.30\fR. В системах Android этот файл обычно называется \fIsepolicy\fR.
+.RE
+.PP
+Если файл политики не указан, она не будет загружена при запуске \fBapol\fR.
+
+.SH ПАРАМЕТРЫ
+.IP "-h, --help"
+Вывести справочные сведения и выйти.
+.IP "--version"
+Вывести сведения о версии и выйти.
+.IP "-v, --verbose"
+Вывести дополнительные информационные сообщения.
+.IP "--debug"
+Включить отладочный вывод.
+
+.SH ОШИБКИ
+Пожалуйста, сообщайте об ошибках через систему отслеживания ошибок SETools, https://github.com/TresysTechnology/setools/issues
```

Comment:
Here and below, use the new location, `https://github.com/SELinuxProject/setools/issues`